packages/react-composites/src/composites/CallComposite/components/ConfigurationpageMicDropdown.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/* @conditional-compile-remove(call-readiness) */
+import React from 'react';
+/* @conditional-compile-remove(call-readiness) */
+import { useTheme, _DevicePermissionDropdownStrings, _DevicePermissionDropdown } from '@internal/react-components';
+/* @conditional-compile-remove(call-readiness) */
+import { useAdapter } from '../adapter/CallAdapterProvider';
+/* @conditional-compile-remove(call-readiness) */
+import { dropDownStyles } from '../styles/LocalDeviceSettings.styles';
+/* @conditional-compile-remove(call-readiness) */
+import { CallCompositeIcon } from '../../common/icons';
+
+/**
+ * @private
+ */
+export interface ConfigurationpageMicDropdownProps {
+  micGrantedDropdown: JSX.Element;
+  micPermissionGranted: boolean;
+}
+
+/**
+ * @private
+ */
+export const ConfigurationpageMicDropdown = (props: ConfigurationpageMicDropdownProps): JSX.Element => {
+  /* @conditional-compile-remove(call-readiness) */
+  const theme = useTheme();
+  /* @conditional-compile-remove(call-readiness) */
+  const adapter = useAdapter();
+  /* @conditional-compile-remove(call-readiness) */
+  const devicePermissionDropdownStringsMicrophone: _DevicePermissionDropdownStrings = {
+    placeHolderText: 'Enable Microphone (required)',
+    actionButtonContent: 'Allow'
+  };
+
+  /* @conditional-compile-remove(call-readiness) */
+  const microphoneBlockedDropdown = (
+    <_DevicePermissionDropdown
+      styles={dropDownStyles(theme)}
+      onClickActionButton={async () => {
+        await adapter.askDevicePermission({ video: false, audio: true });
+        if (props.micPermissionGranted) {
+          adapter.queryMicrophones();
```

Comment:
Is this needed? I would've assumed this would trigger the queries:
![image](https://user-images.githubusercontent.com/2684369/193159377-ec2c09a5-9998-4d0b-b784-cbc318817c42.png)

Reply:
This is for when camera/mic permission is already granted, it will find microphone/camera options and display them on the config screen. Without this when using the calling sample the second time ( already set up permissions the first time so pop up will not be triggered), you wont be able to see any microphone/camera device options.